terraform/istio/install_istio.sh

```diff
@@ -24,8 +24,12 @@ echo "### "
 # Set vars for DIRs
 ISTIO_VERSION="${ISTIO_VERSION:-1.6.2}"
 
-# Set the working directory to our current directory (/sandbox/terraform)
-export WORK_DIR=`pwd`
+# Set the working directory to our current directory (/sandbox/terraform/istio)
+export SCRIPTPATH=$(dirname $(realpath $0))
+cd $SCRIPTPATH
+# WORKDIR and SCRIPTPATH should be the same in our case, 
+# but you may find it more readable to keep both variables around
```

Comment:
Not a big deal, but I'd probably leave this comment out. I put that in for you, not necessarily for the script

Reply:
Haha woops I didn't realize that. Changed :)